boilerplate/app/screens/DemoComponentsScreen/DemoItem.tsx

```diff
@@ -25,9 +19,13 @@ const $demoItemContainer: ViewStyle = {
 }
 
 const $title: TextStyle = {
-  marginVertical: 8,
+  marginTop: 32,
+  paddingTop: 24,
```

Comment:
I'm thinking part of the reason why you have to manage the negative scrollToLocation offset is because of these values. would it make sense to remove this and just add a bottom margin to the DemoItem wrapper?